### ISSUE TITLE
Prediction image  becomes abnormal

### ISSUE THREAD

**Issue by Skydddoogg** [MathiasGruber/PConv-Keras#41]
I use  `pconv_imagenet`  as  pretrained model

The  result abnormally contains blue as shown below. 

<img src="https://sv1.picz.in.th/images/2019/07/25/Koci1t.png" alt="Koci1t.png" border="0">
<img src="https://sv1.picz.in.th/images/2019/07/25/KoccMP.png" alt="KoccMP.png" border="0">

Why does  it become like that  and  how  to fix  it?

**Comment by ghost** [comment]
**My private opinion** 
You can try one to train on the given mask dataset, not random generating.
Of course, you should slightly change some codes.
I had the same result as you, but when I loaded mask dataset and trained model on it, that get me expected results.  If you are more information about this. Let me know. 

Best regards,

**Comment by YuzaChongyi** [comment]
I  have also tried the weights pconv_imagenet but didn‘t get the desired result，and there is always a blank in the upper left of the figure.

**Comment by ghost** [comment]
Let you show some parts of your code that you are thinking ones making abnormal results.
So we can make sure what causes the unexpected on the code.

**Comment by nywang2019** [comment]
@YuzaChongyi my result images also have a white area at the upper left corner,and a black rect block at the upper right corner.

**Comment by nywang2019** [comment]
@Ferruginous-Hawk0215 in fact, i did not modify the code.

**Comment by chongjg** [comment]
> @YuzaChongyi my result images also have a white area at the upper left corner,and a black rect block at the upper right corner.

I also encounter this problem, do you know how to solve this?

**Comment by Skydddoogg** [comment]
Sorry for very very late reply. Here is my situation of getting the abnormal results. I used ADE20K dataset. I modified the segmentation masks of an image by applying the dilation operation, and the dilated masks were used as the input masks of the PConv model. Based on my experiments, these abnormal prediction results occur when the dilated mask is quite big. Hence, to solve this problem, we may need to fine-tune the model with bigger masks.
